service/custom_logs_post_test.go

```diff
@@ -26,41 +23,49 @@ func Test_CustomLogsPostHandler(t *testing.T) {
 	}{
 		{
 			testName:           "ok, minimal",
-			requestBody:        `{}`,
+			requestBody:        `{"log_level":"warn","message":"test","data":{"step_id":"test-step-id","tag":"test-tag"}}`,
 			expectedStatusCode: http.StatusOK,
 			expectedBody:       `{"message":"ok"}` + "\n",
 		},
 		{
-			testName:           "ok, more complex",
-			expectedLogContent: map[string]interface{}{"key1": "value1"},
-			requestBody:        `{"log_level":"info","message":"test message","data":{"key1":"value1"}}`,
-			expectedStatusCode: http.StatusOK,
-			expectedBody:       `{"message":"ok"}` + "\n",
+			testName:           "error, when request body isn't a valid JSON, it retrieves bad reqest error",
+			expectedStatusCode: http.StatusBadRequest,
+			expectedBody:       `{"message":"Invalid request body, JSON decode failed"}` + "\n",
 		},
 		{
-			testName:           "when request body isn't a valid JSON, it retrieves bad reqest error",
+			testName:           "error, when all required param missing",
+			requestBody:        `{}`,
 			expectedStatusCode: http.StatusBadRequest,
-			expectedBody:       `{"message":"Invalid request body, JSON decode failed"}` + "\n",
+			expectedBody:       `{"message":"Invalid request body, please provide log_level"}` + "\n",
+		},
+		{
+			testName:           "error, when log level set",
```

Comment:
NOTE: the issue is not the presence of the `log level`, but the missing `message`,

might `error, missing message` would be more straightforward.